packages/test/test-end-to-end-tests/src/test/gc/gcStateResetInSummaries.spec.ts

```diff
@@ -33,6 +34,7 @@ import { getGCStateFromSummary } from "./gcTestSummaryUtils.js";
 describeNoCompat("GC state reset in summaries", (getTestObjectProvider) => {
 	let provider: ITestObjectProvider;
 	let mainContainer: IContainer;
+	const settings = { "Fluid.ContainerRuntime.Test.CloseSummarizerDelayOverrideMs": 10 };
```

Comment:
It seems like the tests should await for at least time to ensure that the container closes. Otherwise, the test may become flaky.